shell/platform/darwin/ios/framework/Source/profiler_metrics_ios.mm

```diff
@@ -28,9 +30,126 @@
 }
 
 namespace flutter {
+namespace {
+
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG || \
```

Comment:
Given how big this class is getting, consider moving this to `gpu_profiler_metrics_ios.<h|mm>`.

Reply:
The file is 228 lines, *shrug*.  If you feel strongly about it I can move it.  It's an easy change, just takes time with our current system.